### PR TITLE
Leaderboard Time Spinner

### PR DESCRIPTION
**Pull request by mlytvynchuk**

### CODE REVIEW

File: app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java [hunk 1]
```diff
@@ -55,6 +57,9 @@
     private int collection;
     private int recyclerViewId;
     private final int SHOW_SHARING_FRIENDS_CONSENT = 3001;
+    private String ALL_TIME_SCORES;
+    private String WEEKLY_SCORES;
+    private String DAILY_SCORES;
```
Comment (redls) [inline app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java:62]: these variables seem to be used only in this file. So make them final and initialise them directly with the value.

Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java:62]: I have a spinner in xml file with those rows: All time, Weekly, Daily. When the user select the item from the spinner it should change timeSpan of leaderboard(TIME_SPAN_ALL_TIME, TIME_SPAN_WEEKLY, etc).

Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java:62]: To extract these rows: All time, Weekly, Daily I need to extract string-array from xml. That's why I initialized these variables using onCreate method.
Can be improved:
private final String[] timeSpan = getResources().getStringArray(R.array.leaderboard_time);
private final String ALL_TIME_SCORES = timeSpan[0];
private final String WEEKLY_SCORES = timeSpan[1];
OR
private final String ALL_TIME_SCORES = getResources().getStringArray(R.array.leaderboard_time)[0];
private final String WEEKLY_SCORES = getResources().getStringArray(R.array.leaderboard_time)[1];
What do you think?

Comment (redls) [reply app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java:62]: can we do the second option?

Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/LeaderboardScoresFragment.java:62]: I think we can)

**Review by redls — APPROVED** [review]

**Pull request merged**